lob-java-examples/src/main/java/com/lob/examples/AsyncJobExample.java

```diff
@@ -21,7 +21,7 @@
     // This example demonstrates launching multiple requests synchronously and transforming their results
     // once they come in. The only blocking call is the final get().
     public static void main(final String[] args) throws Exception {
-        final String apiKey = "test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc"; // Replace this API key with your own.
+        final String apiKey = "test_7b1960d06d6dfe28d3862b98380b8b0dc93"; // Replace this API key with your own.
```

Comment:
Definition for aapiKey string may be moved to the class BaseExample as static protected constant.